exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -117,13 +122,14 @@ def __init__(
         retry: Optional[str] = DEFAULT_RETRY,
     ):
         self.service_name = service_name
-        self.host_name = host_name
-        self.port = port
-        self.endpoint = endpoint
-        self.protocol = protocol
-        self.url = "{}://{}:{}{}".format(
-            self.protocol, self.host_name, self.port, self.endpoint
-        )
+        self.url = os.environ.get("OTEL_EXPORTER_ZIPKIN_ENDPOINT", DEFAULT_URL)
+
+        url = "{}://{}:{}{}".format(protocol, host_name, port, endpoint)
```

Comment:
I think default value for protocol should be `https`. We can have a special case to use `http` as a default when `endpoint == DEFAULT_ENDPOINT`.

Reply:
+1

I think that makes sense. My only concern is how users can discover the path part of the URL. If a user knows their zipkin endpoints is available at `https://zipkin.corp.com`, would they know or remember to add `:9411` and `/api/v2/spans`? Perhaps documenting the default value in zipkin exporter docs would be enough of an indication?

Reply:
Going with documenting the default value for now.